frb_codegen/src/generator/dart/mod.rs

```diff
@@ -203,10 +203,18 @@ fn generate_import_header(imports: HashSet<&IrDartImport>) -> DartBasicCode {
 
 fn generate_common_header() -> DartBasicCode {
     DartBasicCode {
-        import: "import 'dart:convert';
+        import: if cfg!(feature = "uuid") {
+            "import 'dart:convert';
+            import 'dart:async';
+            import 'package:flutter_rust_bridge/flutter_rust_bridge.dart';
+            import 'package:uuid/uuid.dart';"
```

Comment:
nit: would be great if `if cfg { "import uuid" } else { "" }` so less repeat

Reply:
resolved in 32b0f287c1c6502ba8de3db0873a40f2dbeb3209